src/app/modules/sgid/sgid.service.ts

```diff
@@ -46,9 +48,10 @@ export class SgidServiceClass {
     cookieMaxAge: number
     cookieMaxAgePreserved: number
   }) {
+    this.privateKey = fs.readFileSync(privateKeyPath, { encoding: 'utf8' })
```

Comment:
@LoneRifle not sure if this is the correct encoding for the private key

Reply:
@pregnantboy - anything to add here?